src/hello.hpp

```diff
@@ -0,0 +1,8 @@
+#pragma once
+#include <nan.h>
+// carrots, ex: <nan.h> --> look for header in global
+// quotes, ex: "hello.hpp" --> look for header in location relative to this file
```

Comment:
Good explanation for this at https://stackoverflow.com/a/21594.  @GretaCB - it would be great if you could add a mention to this in the C++ glossary (https://github.com/mapbox/cpp/blob/master/glossary.md).